enclave_wrangler/objects.py

```diff
@@ -32,7 +32,6 @@
     #'content-type': 'application/json'
 }
 DEBUG = False
-# TARGET_CSV_DIR='data/datasets/'
 
 
 class EnclaveClient:
```

Comment:
This `EnclaveClient` is in `enclave_wrangler/objects.py` but one day I think it'll make sense for it to be somewhere else, and re-used.